shell/browser/native_window_views.cc

```diff
@@ -1488,6 +1488,21 @@ bool NativeWindowViews::IsMenuBarVisible() const {
   return root_view_.is_menu_bar_visible();
 }
 
+bool NativeWindowViews::IsSnapped() const {
+#if BUILDFLAG(IS_WIN)
+  // IsWindowArranged() is not a part of any header file.
+  // https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-iswindowarranged
```

Comment:
Apparently it is in a header file, but may have led to crashes on older systems.
https://chromium-review.googlesource.com/c/chromium/src/+/5976961

Seems fine as is though.